Nodejs/Product/Nodejs/Debugger/BinarySerializer.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;

```

Comment:
Can you add the set of files that's copied from elsewhere to a separate folder and add comments to make it clearer that they're shared components?